client/src/api/v2/camera.js

```diff
@@ -39,3 +39,10 @@ export function useOverlays(device) {
 
   return { overlays, setActiveOverlay };
 }
+
+/**
+ * Send Message
```

Comment:
You need to add in what type the `message` parameter should be and what it does. See https://jsdoc.app/about-getting-started.html for info on what to add!